src/eth.rs

```diff
@@ -357,7 +346,7 @@ impl<'d> EthDriver<'d> {
         ),
         EspError,
     > {
-        Self::init_spi_bus::<P>(sclk, sdo, sdi, dma)?;
+        unsafe { gpio_install_isr_service(0) };
```

Comment:
Why is this necessary?

Reply:
It was there before. Not sure why, but I guess it suppresses this error `E (473) gpio: gpio_isr_handler_add(465): GPIO isr service is not installed, call gpio_install_isr_service() first`

Reply:
Pretty sure all chips.